src/main/webapp/WEB-INF/views/editPages/edit_invinterest.html

```diff
@@ -0,0 +1,161 @@
+    <div class="row">
+        <div class="col-xs-12 col-sm-10 col-md-8 col-lg-6 col-lg-offset-3" style="margin-top: 50px;">
+            <div class="panel-card panel-success-card">
+                <div class="text-center"><h3>Edit investment interest</h3></div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-xs-12">
+                            <div class="form-group-card">
+                                <label class="col-md-4 control-label" for="target">Goal</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 25px">
+                                    <select id="target" name="The target" class="form-control">
+                                        <option value="Start-ups and investment projects">Start-ups and
+                                            investment
+                                            projects
+                                        </option>
+                                        <option value="Venture loans to business">Venture loans to business
+                                        </option>
+                                        <option value="Business acquisition">Business acquisition</option>
+                                    </select>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-4 control-label" for="description">Description</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 25px">
+                                            <textarea class="form-control" id="description" style="resize: none;"
+                                                      placeholder="Description"
+                                                      name="description"></textarea>
+                                </div>
+                            </div>
+                            <div class="form-group" style="margin-bottom: 55px">
+                                <label class="col-md-4 control-label" for="category">Category</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 25px">
+                                    <input id="category" name="category" type="text" placeholder="Category"
+                                           class="form-control">
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-4 control-label " for="cost">Cost</label>
+                                <div class="col-xs-12 col-md-6 col-lg-8" style="margin-bottom: 20px">
+                                    <!--<input id="cost" name="cost" type="number" style="width: 300px "  placeholder="Cost"-->

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)